src/components/contact/ContactIconsList.tsx

```diff
@@ -0,0 +1,45 @@
+import { Text, Box, Stack } from '@mantine/core';
+import { IconSun, IconPhone, IconMapPin, IconAt } from '@tabler/icons-react';
+import './ContactIcons.scss';
+
+interface ContactIconProps
+  extends Omit<React.ComponentPropsWithoutRef<'div'>, 'title'> {
+  icon: typeof IconSun;
+  title: React.ReactNode;
+  description: React.ReactNode;
+}
+
+function ContactIcon({
+  icon: Icon,
+  title,
+  description,
+  ...others
+}: ContactIconProps) {
```

Comment:
[No function, only arrow func](https://www.w3schools.com/js/js_arrow_function.asp)
And use props defenition, not props object in func argument

```suggestion
const ContactIcon = (props : ContactIconProps) => {
```